README.md

```diff
@@ -206,6 +206,22 @@ If you want to access STF from other machines, you can add the `--public-ip` opt
 stf local --public-ip <your_internal_network_ip_here>
 ```
 
+## Quick Start
+
+You can try STF in one single mashine instantly.
+
+Requirements:
+- Linux OS (not working on arm images)
```

Comment:
instead of `not working on arm` you should better describe where it DOES run. Does it run on aarch64? RISC-V? amd64? x86? powerpc?

Reply:
Usually I run STF on Ubuntu desktop amd64, but I also tried to run STF on my Pi4. Just for fun, can tiny box operate android devices.
If I run this docker-compose, it does not start for this reason:
`pi@pi:~/workspace/stf$ sudo docker-compose up -d --build`
`Creating network "stf_default" with the default driver`
`Creating volume "stf_rethinkdb-data" with default driver`
`Pulling rethinkdb (rethinkdb:2.3)...`
`2.3: Pulling from library/rethinkdb`
`ERROR: no matching manifest for linux/arm64/v8 in the manifest list entries`

Reply:
> instead of `not working on arm` you should better describe where it DOES run. Does it run on aarch64? RISC-V? amd64? x86? powerpc?

Added describe where it does run.
Commit 89ef4d13d3dd9f670e4000e7f4a82d7332430153